tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "allowJs": true,
-    "target": "ES2017",
+    "target": "ES2018",
```

Comment:
This was bumped to get a successful build. Otherwise, it seems esbuild has issues transforming the async generators.
```
✘ [ERROR] Transforming async generator functions to the configured target environment ("ES2017") is not supported yet
```